app/api/onboarding/register.server.js

```diff
@@ -1,25 +1,24 @@
-export async function register(data) {
```

Comment:
If this file is only having formatting changes then take a new pull and commit again as it was addressed in the recent PR merge.

Reply:
It was throwing Lint error due to which I updated it to Default export, Am I wrong here? Please check